cli/src/main/kotlin/OrtMain.kt

```diff
@@ -171,8 +168,11 @@ class OrtMain : CliktCommand(name = ORT_NAME, invokeWithoutSubcommand = true) {
         printStackTrace = stacktrace
 
         // Make options available to subcommands and apply static configuration.
-        val ortConfig = OrtConfiguration.load(configArguments, configFile)
-        currentContext.findOrSetObject { GlobalOptions(ortConfig, forceOverwrite) }
+        val ortConfig = OrtConfiguration.load(
+            args = mapOf("ort.forceOverwrite" to "$forceOverwrite") + configArguments,
```

Comment:
How does this behave if the option is set in the configuration file? Does it always overwrite (even if the option is not set on the command line) or never?

Reply:
Good point, yes, fixed.

Reply:
Should the order not be the other way round, as if I understand that correctly the value from the config file would now always override the command line option?

```kotlin
putAll(configArguments)
if (forceOverwrite) put("ort.forceOverwrite", "true")
```

Reply:
No, `configArguments` only contains the `-P`-style configuration arguments given on the command line, not any configuration from `config.yml` (that's passed in the next line via `configFile`). And as `--force-overwrite` is being deprecated in favor of `-P ort.forceOverwrite=...`, the latter should override the former syntax. That's why I believe the order to be correct.